fre/make/tests/null_example/test_checkout_null_yaml.py

```diff
@@ -0,0 +1,45 @@
+#tests for the create-checkout step of fre-make, for null_model.yaml
+import os
+from fre import fre
+from pathlib import Path
+from fre.pp import configure_script_yaml as csy
+from click.testing import CliRunner
+import subprocess
+from fre.make import createCheckout
+
+# Set example yaml paths, input directory
+test_dir = Path("fre/make/tests/null_example")
+yamlfile = Path(f"{test_dir}/null_model.yaml")
```

Comment:
I'd recommend keeping all the tests together in fre/make/tests for now. We can move it there and make this 

`TEST_DIR = Path("fre/make/tests")`
`YAMLFILE = Path(f"{TEST_DIR}/null_example/null_model.yaml")`